guide/Digital_Platform_Ambassadors.md

```diff
@@ -0,0 +1,59 @@
+## Digital Platform Ambassadors
+ 
+The DPA has 12 inaugural members. Ahmad Nassri and Steve Fisher will oversee the smooth administration and a clear agenda. Its initial focus is on front-end consistency of the central TELUS experience. As needs change other members may be invited and ceremonies may change.
+
+- Mobility
```

Comment:
@jackreeves 
Would it be clearer to just add a link to the [GitHub team](https://github.com/orgs/telus/teams/dpa-developers) for DPA?
If we include a hard-coded list of members here, we will have to maintain it ...